src/systems/ec/implementations/react/templates/pages/research-area-hub/src/ResearchAreaHubPage.jsx

```diff
@@ -293,55 +294,47 @@ const ResearchAreaHubPage = ({ siteHeader, footer, template }) => (
 
               <div className="ecl-row ecl-u-mt-l">
                 <div className="ecl-col-12 ecl-col-md-6">
-                  <article className="ecl-u-border-bottom ecl-u-border-color-grey-15 ecl-u-pv-m ecl-u-height-100 ecl-u-box-sizing-border">
-                    <div className="ecl-u-type-prolonged-m ecl-u-type-bold">
-                      <Link
-                        href="/example"
-                        variant="standalone"
-                        label="Horizon 2020"
-                      />
-                    </div>
-                    <p className="ecl-u-type-paragraph ecl-u-type-color-grey-100 ecl-u-mt-xs ecl-u-mb-none">
-                      Calls for proposal related to marine resources are spread
-                      across numerous schemes in Horizon 2020. This page lists
-                      them and explains how you can apply.
-                    </p>
-                  </article>
+                  <ContentItem
+                    className="ecl-u-height-100 ecl-u-box-sizing-border"
+                    title={{
+                      href: '/example',
+                      label: 'Horizon 2020',
+                    }}
+                    description={{
+                      label:
+                        'Calls for proposal related to marine resources are spread across numerous schemes in Horizon 2020. This page lists them and explains how you can apply.',
+                    }}
+                  />
                 </div>
 
                 <div className="ecl-col-12 ecl-col-md-6">
-                  <article className="ecl-u-border-bottom ecl-u-border-color-grey-15 ecl-u-pv-m ecl-u-height-100 ecl-u-box-sizing-border">
-                    <div className="ecl-u-type-prolonged-m ecl-u-type-bold">
-                      <Link
-                        href="/example"
-                        variant="standalone"
-                        label="European structural and investment funds"
-                      />
-                    </div>
-                    <p className="ecl-u-type-paragraph ecl-u-type-color-grey-100 ecl-u-mt-xs ecl-u-mb-none">
-                      Marine research related calls may be found in this
-                      programme, particularly in the European maritine and
-                      fishieries fund.
-                    </p>
-                  </article>
+                  <ContentItem
+                    className="ecl-u-height-100 ecl-u-box-sizing-border"
+                    title={{
+                      href: '/example',
+                      label: 'European structural and investment funds',
+                    }}
+                    description={{
+                      label:
+                        'Marine research related calls may be found in this programme, particularly in the European maritine and fishieries fund.',
+                    }}
+                  />
                 </div>
               </div>
 
               <div className="ecl-row">
```

Comment:
Every second row below have an `ecl-u-mt-m` classes, this one doesn't. Personally, I don't understand why a margin-top is defined on the following rows. This creates a larger visual gap at the top on all following rows.

Reply:
I agree, these margins are not needed